README.md

```diff
@@ -1,236 +1,192 @@
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Prerequisites](#prerequisites)
-  - [Setup](#setup)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 World Weather App ](#-workd-weather-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 World Weather App <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+**World Weather App** display the weather of all states of the selected country.
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
+1. ✅ **React**
+1. ✅ **Redux**
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+### Tech Stack <a name="tech-stack"></a>
 
 <details>
-  <summary>Client</summary>
+  <summary>Markup</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li>HTML</li>
+    <li>MD markup</li>
   </ul>
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>Style</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li>CSS</li>    
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+<summary>Dynamic</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li>JavaScript</li>
+    <li>React</li>
+    <li>WepPack</li>
   </ul>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+-  **React-based project**
+-  **Responsive**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+<!-- Live Demo -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- ✅ Live Demo [Link](https://world-weather-app.onrender.com)
+- ✅ Video [Link](https://drive.google.com/file/d/1wp1H1ncSPwZVLj3nHGlbDQwaohRcAOZn/view?usp=sharing)
 
```

Comment:
Good job working on the project requirements. But your video is not opening up and it says in the project requirements that you are supposed to use loom to record a video presentation of your project.

This is what Im seeing when i open the video link:

![Screenshot 2023-08-04 at 7 12 39 AM](https://github.com/Alibaba2023/world-weather-app/assets/79506105/cb9867b4-fef0-441e-8bca-549c8cc20ee5)
It would be nice to record another video based on the following requirements;

Video presentation

Provide feedback about the video presentation in the code review summary.

The student recorded a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
The video contains a demo of the project, highlighting all the project features.
The video contains a highlight about a piece of the code the student finds interesting or is proud of making.


Note: It's recommended that students use [Loom](https://www.loom.com/) to record to video but they can use any tool they want as long as they share an online link of the uploaded video.